(DOCSP-8943): Realm Event Logs (#282)
mongodb/docs-realm: source/logs/sync.txt

=========
Sync Logs
=========

.. default-domain:: mongodb

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 2
   :class: singlecol

Overview
--------

{+service+} creates sync logs whenever a user interacts with Sync. This
includes writing data to {+backend+} from clients, reading changes from

> {+backend+} to clients, as well as any time a connection or session starts

Reviewer: Parallel structure (plus avoid muddy phrases like "as well as"):
```suggestion
{+backend+} to clients, and starting or ending a connection.
```

> or ends.

Reviewer: With above:
```suggestion
```

events include downloaded changes from all clients on all partitions as
well as uploaded changes to data from MongoDB Atlas. Any sync event that
does not include a user ID is part of the MongoDB Atlas data
synchronization process. You may see a large number of MongoDB Atlas
synchronization events when you initialize sync if your cluster already
contains a large amount of data.

Fields
~~~~~~

.. list-table::
   :header-rows: 1
   :widths: 20 50

   * - Field
     - Description

   * - Logs
     - Debugging information about the operation. Includes the schema
       version on client and server, the number of changes, and whether
       or not conflict resolution was required in order to incorporate
       the operation into the server's copy of the data.

   * - Partition
     - Which :term:`partition` changed as a result of the operation.

   * - Write Summary
     - Includes information about the data that changed, such as:

       - The type or table of data that changed.

       - The operation used to change data: insertion, deletion, update, or replace.

       - The primary key of the data that changed.

   * - SDK

>      - The SDK used to send the request. Potential values include any :doc:`{+service-short+} Client SDK </procedures/init-realmclient>`. For a browser, this is ``<browser> <SDK version>``.

Reviewer: ```suggestion
     - The SDK used to send the request. Potential values include any :doc:`{+service-short+} Client SDK </ios>`. For a browser, this is ``<browser> <SDK version>``.
```
